compiler/damlc/tests/interfaces-data-deps/interface-implementation/daml/Asset.daml

```diff
@@ -0,0 +1,86 @@
+-- Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module Asset where
+
+import Token
+
+import Daml.Script
+
+import DA.Assert
+
+template Asset
```

Comment:
Should we also test that an interface implementation can be used across-data-deependencies?

Reply:
that's a very good point, will do!